packages/flutter/lib/src/services/system_chrome.dart

```diff
@@ -221,6 +221,9 @@ class SystemChrome {
   /// The `orientation` argument is a list of [DeviceOrientation] enum values.
   /// The empty list causes the application to defer to the operating system
   /// default.
+  ///
+  /// **Important:** This setting will only be respected on iPad if multitasking is disabled for your app.
```

Comment:
Please don't use `**Important:**`. Just add what needs to be documented as a regular paragraph. You may add a heading `## Limitations` or something like that.